opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -111,6 +112,25 @@ func resourceOpennebulaVirtualMachine() *schema.Resource {
 				Computed:    true,
 				Description: "Current LCM state of the VM",
 			},
+			"desired_state": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     false,
```

Comment:
false is not a valid state. If you do not provide the state terraform will use "false" while  you expect one of these: "running", "stopped", "suspended", "undeployed", "poweroff"

Reply:
Would it be an adequate solution if we added "ignore" as default and not support hold or pending?